packages/react/src/ActionList/shared.ts

```diff
@@ -53,6 +53,9 @@ export type ActionListItemProps = {
    */
   _PrivateItemWrapper?: React.FC<React.PropsWithChildren<MenuItemProps>>
   className?: string
+  groupId?: string
+  renderItem?: (item: React.FC<React.PropsWithChildren<MenuItemProps>>) => React.ReactNode
+  handleAddItem?: (item: React.FC<React.PropsWithChildren<MenuItemProps>>) => void
```

Comment:
Wanted to highlight this as something I'm unsure of? @joshblack I had to add these here because of the props I added on the item to satisfy the type errors